src/Authenticator/HttpBasicAuthenticator.php

```diff
@@ -66,7 +66,7 @@ public function getUser(ServerRequestInterface $request)
 
         return $this->_identifier->identify([
             IdentifierInterface::CREDENTIAL_USERNAME => $username,
-            IdentifierInterface::CREDENTIAL_PASSWORD => $password
+            IdentifierInterface::CREDENTIAL_PASSWORD => $password,
```

Comment:
Why?

Reply:
Multiline trailing comma. We should have a sniffer for this, though IMO.

Reply:
OK, I can see why it's a good thing. More readable git diffs for instance :)